metadata/metadata-modeshape/src/main/java/com/thinkbiganalytics/metadata/modeshape/BaseJcrProvider.java

```diff
@@ -355,12 +408,79 @@ public void deleteById(PK id) {
         };
     }
 
+    protected StringBuilder startBaseQuery(String... props) {
+        String alias = getEntityAlias();
+        StringBuilder bldr = new StringBuilder("SELECT ");
+        if (props.length == 0) {
+            bldr.append(alias).append(".* ");
+        } else {
+            boolean start = true;
+            for (String prop : props) {
+                if (! start) {
+                    bldr.append(", ");
+                    start = false;
+                }
+                bldr.append(alias).append(".").append(prop);
+            }
+        }
+        return bldr.append(" FROM [").append(getNodeType(getJcrEntityClass())).append("] AS ").append(getEntityAlias()).append(" ");
+    }
+    
+    protected String getEntityAlias() {
+        return "e";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method and declare a constant for this value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3400)